### DISCUSSION TITLE
Training "VCTK + new data" vs fine-tuning with "new data"?

### DISCUSSION THREAD

**Discussion by rtruszkowski** [coqui-ai/TTS#2819] [General Q&A]
I have been getting okay results just by fine-tuning yourtts with a few minutes of my voice, getting okay results after ~100 epochs. I saw previous posts discussing training from scratch using the original VCTK set + my voice data. This is an enormous amount of data, and I will never be able to train this well given my resources... Or is it fine-tuning, but original VCTK + new data?

Can someone help me understand the difference between these two approaches? I could get a Colab notebook to try this out, but I don't know if it's worth it.

Thanks!

**Comment by Lguyogiro** [comment]
My understanding is that it is fine-tuning with some dataset plus your custom voice data, as explained [here](https://github.com/coqui-ai/TTS/discussions/2708#discussioncomment-6272686).

By the way, could you share your recipe and procedure for getting "okay results" with only a few minutes of your voice? I have about 30 minutes and my results are poor (e.g. the voice sounds right, but is just producing garbled, mostly unintelligible speech). 

I appreciate any insight into your current training procedure!

**Reply by truszko1** [reply]
How many epochs you run? And which model? I fine tuned the yourtts model using the recipe they have shared here in the recipes dir. the only functional changes I made were batch sizes, paths etc.

**Reply by Lguyogiro** [reply]
ran it for 1,000 epochs, also followed the recipe, fine-tuning on around 30 minutes of speech.

**Reply by xyyimian** [reply]
@truszko1 will appreciate if you could share your training config file. I met the same problem that the voice is similar but I can not hear any words. Thanks!
